### ISSUE TITLE
cropping: `colSeparator` keeps growing

### ISSUE THREAD

**Issue by bertsky** [mjenckel/ocrd_anybaseocr#28]
Currently, `ocrd-anybaseocr-crop` overwrites the `colSeparator` parameter it is passed by multiplying it each time it processes an input file, as can be seen [here](https://github.com/mjenckel/ocrd_anybaseocr/blob/33cc8186fc7147565257ed3b6e6d0d3dec3f5272/ocrd_anybaseocr/cli/ocrd_anybaseocr_cropping.py#L503).

I don't know if that actually does any harm, but I think it's not correct. And it creates annotations like this after 100 or so files:

```XML
<pc:MetadataItem type="processingStep" name="preprocessing/optimization/cropping" value="ocrd-anybaseocr-crop">
  <pc:Labels>
    <pc:Label value="1304301559452385791083860667921945049512670652676283396137822601561251067109877615124435106720588263854021939843988814105709049577629659822293447714861872273775414540198337591612107674459312338931170881599710958847590103069139491522272792245308087798808372170911863890515942476730728360803222081593683116980145571758080000000000000000000000000000000" type="colSeparator"/>
```

**Comment by kba** [comment]
>  1304301559452385791083860667921945049512670652676283396137822601561251067109877615124435106720588263854021939843988814105709049577629659822293447714861872273775414540198337591612107674459312338931170881599710958847590103069139491522272792245308087798808372170911863890515942476730728360803222081593683116980145571758080000000000000000000000000000000

:-O

**Comment by mahmed1995** [comment]
The issue has been resolved in commit 8b1758485866a206ef14f927377dac1aff1a6dac
